package.json

```diff
@@ -23,7 +23,7 @@
   ],
   "license": "MIT",
   "homepage": "https://github.com/denar90/iframe-defer-element#readme",
-  "bugs": "https://github.com/brunch/denar90/iframe-defer-element",
+  "bugs": "https://github.com/denar90/defered-iframe-element/issues",
```

Comment:
I think here should be `iframe-defer-element`, instead of `defered-iframe-element`, but `brunch` - nice find 👍  😄